service/matching/service.go

```diff
@@ -55,7 +55,7 @@ func NewConfig() *Config {
 		UpdateAckInterval:               10 * time.Second,
 		OutstandingTaskAppendsThreshold: 250,
 		MaxTaskBatchSize:                100,
-		MinTaskThrottlingBurstSize:      10000,
+		MinTaskThrottlingBurstSize:      1,
```

Comment:
test change or final value?

Reply:
Final value. I did some more research to understand this and having a high burst size would allow a burst of say 10000 requests in an instant which is not what we want. If you're interested, I can explain more offline.